rbac/management/management/commands/init_tenant.py

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
We can keep this removed, right? Since we've now pulled out django tenant schemas, this may cause issues with its imports and references to DTS functions.